src/components/SecretsDeleteModal/SecretsDeleteModal.scss

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '../../scss/vars';
```

Comment:
this isn't being used, but we *should* use the Carbon design tokens for colour, spacing, etc.
`@import '~carbon-components/scss/globals/scss/vars';` to access them

Reply:
should i use the variables declared in `../../scss/vars` or use `@import '~carbon-components/scss/globals/scss/vars';` ???

Reply:
`@import '~carbon-components/scss/globals/scss/vars';`